hack/update-checksum-windows.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+YAMLS_DIR="${WORK_DIR}"/../build/yamls
+WINDOWS_DIR="${YAMLS_DIR}"/windows
+WINDOWS_YAMLS="${YAMLS_DIR}/antrea-windows.yml ${YAMLS_DIR}/antrea-windows-containerd.yml ${YAMLS_DIR}/antrea-windows-containerd-with-ovs.yml"
+
+checksum_windows_config=$(cat ${WINDOWS_DIR}/base/conf/antrea-agent.conf ${WINDOWS_DIR}/base/conf/antrea-cni.conflist | sha256sum | cut -d " " -f 1)
```

Comment:
can check if files exists before calculating checksum, some files might get removed/renamed etc in future, similar for others for error handling etc
if [ -f "${WINDOWS_DIR}/base/conf/antrea-agent.conf" ] && [ -f "${WINDOWS_DIR}/base/conf/antrea-cni.conflist" ]; then

Reply:
the script should definitely fail if a file is missing though, it should not silently ignore the missing file